applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_process_utilities.cpp

```diff
@@ -0,0 +1,126 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "custom_processes/apply_excavation_process.h"
+#include "custom_processes/apply_k0_procedure_process.h"
+#include "custom_utilities/process_utilities.h"
+#include "testing/testing.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(GetModelPartsFromSettings_SingleModelPart, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    Model model;
+    model.CreateModelPart("Main");
+
+    Parameters settings(R"(
+        {
+            "model_part_name": "Main"
+        })");
+
+    auto model_parts = ProcessUtilities::GetModelPartsFromSettings(model, settings, "TestProcess");
```

Comment:
I was wondering whether we can make this variable `const`:
```suggestion
    const auto model_parts = ProcessUtilities::GetModelPartsFromSettings(model, settings, "TestProcess");
```

Reply:
we can ;) added